src/main/kotlin/no/nav/familie/ef/sak/patchstatistikk/PatchStatistikkController.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.familie.ef.sak.patchstatistikk
+
+import no.nav.familie.ef.sak.behandling.BehandlingRepository
+import no.nav.familie.ef.sak.behandling.domain.Behandling
+import no.nav.familie.ef.sak.behandlingsflyt.task.BehandlingsstatistikkTask
+import no.nav.familie.prosessering.internal.TaskService
+import no.nav.security.token.support.core.api.Unprotected
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/patch-statistikk")
+@Unprotected
+class PatchStatistikkController(private val taskService: TaskService, private val behandlingRepository: BehandlingRepository) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    @PostMapping("/henlagt")
+    fun patchHenlagteBehandlingerTilStatistikk(@RequestBody liveRun: LiveRun) {
+        val behandlinger: List<Behandling> = finnBehandlingerSomErHenlagt().filterNot { it.erMigrering() }
+        logger.info("Skal patche ${behandlinger.size} henlagte behandlinger til datavarehus med ferdigstilt/henlagt-resultat")
+
+        val behandlingsstatistikkPayloads = behandlinger.map {
+            BehandlingsstatistikkTask.opprettHenlagtTask(
+                behandlingId = it.id,
+                hendelseTidspunkt = it.sporbar.endret.endretTid,
+                gjeldendeSaksbehandler = it.sporbar.endret.endretAv
+            )
+        }
+        if (liveRun.skalPersistere) {
+            logger.info("Persisterer behandlingsstatistikk-tasks for henlagte behandlinger")
+            behandlingsstatistikkPayloads.forEach {
+                taskService.save(it)
```

Comment:
Hvis vi ikke setter `@Transactional` på dette så vil hver save være en egen transaction tror jeg, då burde man kunne ha en try catch rundt hver save her som sjekker på duplicate, hvis duplicate så logg warning eller noe slik :)

Reply:
Kommer ikke til å få "duplicate" siden "now" og endretTid aldri er lik i payloaden. Men DVH godtar litt "for mye" data per behandling - de leser bare den med høyest teknisk tid